.circleci/config.yml

```
@@ -6,20 +6,20 @@ orbs:
 executors:
   golang-exec:
     docker:
-      - image: cimg/go:1.18
+      - image: cimg/go:1.19
 
 jobs:
   test:
     working_directory: /home/circleci/go/src/github.com/fairwindsops/gemini
 
     docker:
-      - image: cimg/go:1.18
+      - image: cimg/go:1.19
     steps:
       - checkout
       - run: |
-          go get -u gotest.tools/gotestsum
-          mkdir -p /tmp/test-results/go
-          gotestsum --junitfile /tmp/test-results/go/results.xml -- ./pkg/... -cover -covermode atomic -coverpkg=./... -coverprofile=coverage.txt
+          curl -L "https://github.com/gotestyourself/gotestsum/releases/download/v1.8.1/gotestsum_1.8.1_linux_amd64.tar.gz" > /tmp/gotestsum.tar.gz
+          tar -xvf /tmp/gotestsum.tar.gz -C /tmp/
+          mv /tmp/gotestsum /usr/local/bin/
```

Looks like we lost the `gotestsum` command here?

```suggestion
          go install gotest.tools/gotestsum@latest
          mkdir -p /tmp/test-results/go
          gotestsum --junitfile /tmp/test-results/go/results.xml -- ./pkg/... -cover -covermode atomic -coverpkg=./... -coverprofile=coverage.txt

```